tests/vcr_cassettes/2sa_flow_valid_device.yml

```diff
@@ -775,9 +884,10 @@ interactions:
     status: {code: 200, message: OK}
 - request:
     body: '{"batch": [{"resultsLimit": 1, "query": {"filterBy": {"fieldName": "indexCountID",
-      "fieldValue": {"type": "STRING_LIST", "value": ["CPLAssetByAddedDate"]}, "comparator":
+      "fieldValue": {"type": "STRING_LIST", "value": ["CPLAssetByAssetDateWithout"]}, "comparator":
```

Comment:
Should that be `CPLAssetByAssetDateWithoutHiddenOrDeleted` instead?

Reply:
adapted - did not affect test

Reply:
> adapted - did not affect test

I was curious why different string did not cause test failure in the first place and you confirmed that change did not affect test as well. Do you know why? IIUC vcr matches request to the fields in file.

Reply:
I am new to this but it seems by default only 'method', 'scheme', 'host', 'port', 'path', 'query' are used to match requests, so not the body. When I tried to enable 'body' all tests needed to be adapted, and since a lot of the cassette files are handcrafted it is not an easy job, so I went for minimal impact.